src/backend/nodes/entity.py

```
@@ -0,0 +1,239 @@
+            "entity_ids": entity_ids,
+        }
+
+    def exec_fallback(self, prep_res, exc):
+        print(f"EntityResolver failed: {exc}")
+        return {
+            "entities": [],
+            "entity_map": {},
+            "knowledge_hints": {},
+        }
+
+    def post(self, shared, prep_res, exec_res):
```

Ruff (0.14.10)

Unused method argument: `prep_res`

(ARG002)